api/HTMLMediaElement.json

```diff
@@ -2972,7 +2980,8 @@
               "version_added": "20"
             },
             "firefox_android": {
-              "version_added": true
+              "prefix": "moz",
+              "version_added": "20"
```

Comment:
Coming across this entry I saw a bunch of things wrong so I've sent https://github.com/mdn/browser-compat-data/pull/7518 and https://github.com/foolip/mdn-bcd-collector/pull/831.

We haven't confirmed that the prefixed data is correct here to begin with, but let's assume that and update it if we find the desktop data to be wrong later.